docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,42 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.6 - November 28, 2023
+
+### New Features
+
+- Live migration
+
+  - Support for [live migration](../migrate/live-migrate/) from Oracle databases (with the option of [fall-back](../migrate/live-fall-back/)) {{<badge/tp>}}, using which you can fall back to the original source database if an issue arises during live migration.
+
+  - Various commands that are used in live migration workflows (including fall-forward) have been modified. Refer to [live migration](../migrate/live-migrate/) documentation for more details.
```

Comment:
@aishwarya24 We thought it would be good to summarize the changes in commands here-  
The modifications are: 

1. `yb-voyager fall-forward setup ...` -> `yb-voyager import data to source-replica ...`
2. `yb-voyager fall-forward synchronize ...` -> `yb-voyager export data from target ...`
3. `yb-voyager cutover initiate ...` -> `yb-voyager initiate cutover to target/source-replica ...`
4. We're moving away from the "fall-forward database" terminology in favour of "source-replica database".

Also, can you pls also link to fall-forward migrate docs?